CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 * [CHANGE] `trace_id` field in log files has been renamed to `traceID`. #2518
 * [CHANGE] Slow query log has a different output now. Previously used `url` field has been replaced with `host` and `path`, and query parameters are logged as individual log fields with `qs_` prefix. #2520
 * [CHANGE] Experimental WAL: WAL and checkpoint compression is now disabled. #2436
+* [CHANGE] Updated weaveworks/common dependency to `v0.0.0-20200422083114-5790e7482ff8`. The latest changes in weaveworks/common/server allow to speficy TLS options for the server. #2535
```

Comment:
We usually don't add any entry in the changelog for any change which is not user-facing. I would suggest to remove this.